Cargo.toml

```diff
@@ -26,7 +31,7 @@ vec_map = "0.8.2"
 smallvec = "1.8.0"
 arbitrary = { version = "1.2.3", features = ["derive"], optional = true }
 alloy-primitives = { version = "1.0.0" }
-
+context_deserialize = { version = "0.1", optional = true }
```

Comment:
Rather than depending on 0.1, I think we should release 0.2 first with all the impls deleted

Otherwise we could end up with overlapping impls when the `milhouse` feature is enabled on context_deserialize 0.1?